services/brig/src/Brig/API/Internal.hs

```diff
@@ -124,7 +123,9 @@ mlsAPI =
     :<|> mapKeyPackageRefsInternal
 
 accountAPI :: ServerT BrigIRoutes.AccountAPI (Handler r)
-accountAPI = Named @"createUserNoVerify" createUserNoVerify
+accountAPI =
+  Named @"createUserNoVerify" createUserNoVerify
+    :<|> Named @"createUserNoVerifySpar" createUserNoVerifySpar
```

Comment:
When will this new endpoint be called? Is there a test for it?

Reply:
This is called when creating users via `createBrigUserSAML`, from `createValidScimUser` in `Spar/Scim/User.hs` and is tested through those integration tests, as well as through `createSamlUserWithId` from `Spar/App.hs`.